packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -96,6 +96,7 @@ registerFragment(`
     group {
       _id
       name
+      organizerIds
```

Comment:
This is so that the frontend has all the info it needs to do the permissions check